mock/client/sendMessage.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+const config = require('./config')
```

Comment:
Think about this code as documentation, we need a lot more comments here that explain what's going on.